server/routes/statistics.js

```diff
@@ -78,12 +82,42 @@ module.exports = function (config) {
 				dateRanges = getDateRangesByMode(query.dateFrom, query.dateTo, query.mode);
 				dateRanges.forEach(function (dateRange) {
 					// dateRange: { dateFrom: db.Date, dateTo: db.Date } with dateRange query for result
-					result[mode.getDisplayedKey(dateRange.dateFrom)] = null;
+					result[mode.getDisplayedKey(dateRange.dateFrom)] = {};
 				});
 
 				return result;
 			});
 		},
+		'get-flow-roles-operators-data': function (query) {
+			return flowQueryHandlerOperators.resolve(query)(function (query) {
+				var dateRanges, result = {}, mode, page = Number(query.page)
+				  , finalResult = {}, itemsCnt = 0, currentPage = 1;
+				mode = modes.get(query.mode);
+				dateRanges = getDateRangesByMode(query.dateFrom, query.dateTo, query.mode);
+				dateRanges.forEach(function (dateRange) {
+					// dateRange: { dateFrom: db.Date, dateTo: db.Date } with dateRange query for result
+					result[mode.getDisplayedKey(dateRange.dateFrom)] = {};
+				});
+
+				result = reduceOperators(result, query);
```

Comment:
If I read correctly result is just hash of empty objects (as filled at line 99), and it is passed to `reduceOperators`, which per documentation expect object that's is result of `calculateStatusEventsSums`.
Will it be filled after #1758 is ready? If it's so, maybe let's put some TODO comment here, at least it's totally not clear to me.